SourceryTests/Models/TypeNameSpec.swift

```diff
@@ -130,6 +133,16 @@ class TypeNameSpec: QuickSpec {
                 }
             }
 
+            context("given closure type inside generic type") {
+                it("reports closure correctly") {
+                    expect(TypeName("Foo<() -> ()>").isClosure).to(beFalse())
+                    expect(TypeName("Foo<(String) -> Bool>").isClosure).to(beFalse())
+                    expect(TypeName("Foo<(String) -> Bool?>").isClosure).to(beFalse())
+                    expect(TypeName("Foo<(Bar<String>) -> Bool)>").isClosure).to(beFalse())
+                    expect(TypeName("Foo<(Bar<String>) -> Bar<Bool>)>").isClosure).to(beFalse())
```

Comment:
ditto, brackets are not balanced, not sure if that's on purpose?

Reply:
thanks for pointing out, it was a typo, fixed now.